recipe/meta.yaml

```diff
@@ -30,12 +29,7 @@ test:
 
 about:
   home: https://github.com/Jhsmit/PymageJ
-  license: GPL2
-  license_family: GPL
-  # Not currently included, but will be with referenced PR.
-  #
-  # https://github.com/Jhsmit/PymageJ/pull/2
-  #
+  license: GPL-2.0-or-later
   license_file: LICENSE.txt
```

Comment:
Should this still be `LICENSE.txt`? It appears to be called [`LICENSE.md` in the repo]( https://github.com/Jhsmit/PymageJ/blob/0.2.1/LICENSE.md ). Happy to update this if needed :)

Reply:
Indeed license.md, thanks.

Reply:
I've updated to license.md